packages/livechat/src/widget.js

```diff
@@ -81,7 +81,7 @@ const updateWidgetStyle = (isOpened) => {
 	}
 
 	if (isOpened) {
-		widget.style.left = smallScreen ? '0' : 'auto';
+		widget.style.left = smallScreen && widget.dataset.state !== 'triggered' ? '0' : 'auto';
```

Comment:
Could you save this value to avoid repetition? `cons isTriggered = widget.dataset.state === 'triggered'`